addons/edi_stock/models/edi_move_cancellation_report_document.py

```diff
@@ -20,6 +20,13 @@ class StockMove(models.Model):
         index=True,
     )
 
+    cancelled_by = fields.Many2one("res.users", string="Cancelled By", readonly=True)
+
+    def _action_cancel(self):
+        User = self.env["res.users"]
+        self.cancelled_by = User.browse(self.env.real_uid)
```

Comment:
Need to make sure this works when self contains multiple moves. Probably safer to use `write`.

Reply:
Agreed. Aight this is my last comment before an 11 hour flight. Gl hf.

Reply:
changed